kratos/elements/levelset_convection_element_simplex.h

```diff
@@ -189,7 +189,7 @@ class LevelSetConvectionElementSimplex
             const double norm_vel = norm_2(vel_gauss);
             array_1d<double, TNumNodes > a_dot_grad = prod(DN_DX, vel_gauss);
 
-            const double tau_denom = std::max(dyn_st_beta *dt_inv + 2.0 * norm_vel / h + beta*div_v,  1e-2);
+            const double tau_denom = std::max(dyn_st_beta *dt_inv + 2.0 * norm_vel / h + std::abs(beta*div_v),  1e-2);
```

Comment:
Note that beta is hardcoded to be 0 in line 165... I don't see the use of this change.